grails-app/services/org/pih/warehouse/core/UserService.groovy

```diff
@@ -195,7 +195,7 @@ class UserService {
     Boolean hasRoleApprover(User u) {
```

Comment:
I think this should be also renamed, since we will have two "approver" roles now

Reply:
>  Add a translation for the new role type and rename the translation for approver
I didn't do that because I can't see any translations for the previous `ROLE_APPROVER`